libcxx/include/__ranges/drop_view.h

```diff
@@ -90,6 +90,10 @@ namespace ranges {
       requires (!(__simple_view<_View> &&
                   random_access_range<const _View> && sized_range<const _View>))
     {
+      if constexpr (random_access_range<const _View> && sized_range<const _View>) {
+        auto __dist = ranges::distance(__base_);
+        return ranges::begin(__base_) + std::min<range_difference_t<_View>>(__count_, __dist);
```

Comment:
I think you don’t need to explicit template parameter here because they are the same type.  If you want clarity, I would replace the ‘auto’ above with range_difference_t

Reply:
Yep, got it.